index.src.html

```diff
@@ -2819,6 +2842,18 @@
     The {{VideoFrame/duration}} getter steps are to return
     {{VideoFrame/[[duration]]}}.
 
+: <dfn attribute for=VideoFrame>hasAlpha</dfn>
+:: Whether or not the {{VideoFrame}} has an alpha channel. If this is `false`,
+    the frame is considered to be opaque, and must be rendered as such.
```

Comment:
I'd expand more on the contrasting false vs true behavior. Something like

When false 
- the frame must be treated as fully opaque (for example, by any method which takes a CanvasImageSource)
- for planar formats, an alpha plane must not be present
- for interleaved formats, blocks of samples will contain an 'X' channel where the alpha channel may otherwise have been. The position of the 'X' channel is defined by [[format]]. The values of the 'X' channel are undefined and must be ignored when rendering.

When true 
- the frame must be considered transparent for the pixels and degrees indicated by the alpha values.
- for planar formats, the alpha plane must appear as an additional plane following the non-alpha planes defined by [[format]]
- for interleaved formats, the alpha channel will appear at the 'A' position as defined by [[format]]

Just a draft, could probably use additional editing.